src/element/textElement.ts

```diff
@@ -863,6 +863,7 @@ const VALID_CONTAINER_TYPES = new Set([
   "ellipse",
   "diamond",
   "arrow",
+  "embeddable",
```

Comment:
was this intentional? I don't think it makes sense for embeddable elements to support binding text.

Reply:
It is intentional. If the embeddable has a bound text element that will be used as the alt-text when exporting to PNG or SVG. This could be enhanced further in the future to be less of a hidden feature... Right now there are two ways you can edit the text:
1) by creating the alt-text in a separate text element, then binding it. If you want to change it, you need to unbind, change the text, and bind again.
2) you can add text like to a rectangle when the embeddable is empty. In this case the workflow is to create an empty embeddable, add text like you would to a rectangle, then set the link.

Reply:
To me the whole use case of using alt text for export seems niche in that it doesn't add much value and very few people would use it _even_ if it was actually discoverable.

But if we are to support alt text, we should implement it in a proper way. Reusing bound text is not only undiscoverable, but also forces these weird workflows (the [1] doesn't work for me when the embed already has a url), and may/will have unintended consequences. We'd also have to maintain binding logic to work with embeds just for this.

So let's decide if we absolutely need alt text. I'm for dropping it, but if you really insist, then either:

1. add UI for it in the link editor
2. use some syntax which would still be undiscoverable but has some logic/precedent behind it. I propose using the CommonMark syntax for link titles: `[text "title"](url)` https://spec.commonmark.org/0.30/#link-title

I'd start with (2) especially because I'm not bought on the benefit/cost ratio yet.

Reply:
but just to clarify, my first go to would be dropping support altogether.

Reply:
We can drop it. I also don't think many people would use it, especially as it is not even obvious the feature is there. I am ok to drop it.

Reply:
removed in [`7f18eda` (#6691)](https://github.com/excalidraw/excalidraw/pull/6691/commits/7f18eda626a17d197313ae31d31c4dd5e7e7fcfb), please check if I missed anything 🙏